src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/Transform.cs

```diff
@@ -204,7 +204,7 @@ public Hashtable PropagatedNamespaces
                 CanonicalXmlNodeList namespaces = null;
                 if (reference != null)
                     namespaces = reference._namespaces;
-                else if (signedXml._context != null)
+                else if (signedXml?._context != null)
```

Comment:
@bartonjs @krwq I can confirm netfx also throws a `NullReferenceException`. I appear to have found and inadvertently fixed a framework bug. I'll revert this change later today. How do I raise the issue against .Net Framework?